packages/common/src/models/Track.ts

```diff
@@ -44,6 +44,11 @@ export type RemixOf = {
   tracks: Remix[]
 }
 
+export type CoverAttribution = {
+  original_song_title?: string
```

Comment:
i would think that these would be mandatory (but could be empty strings) and then the cover attribution as a whole is optional